tools/objc/interface.py

```diff
@@ -66,20 +66,20 @@ def __load_interfaces(self, config, package, type):
     @staticmethod
     def __find_class_hierarchy(config, package, type):
         base = "GlyCommon"
```

Comment:
I see. Currently GlyCommon must be base class. A lot of things depend on it. Making it configurable just here wont be enough.